qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -33,3 +33,17 @@
                 clean_cmd = del /f /q
                 tmp_dir = %TEMP%
                 python_bin = python2.7
+        - hotplug_various_chardev:
+            type = virtio_serial_various_chardev_hotplug
+            serials = ""
+            start_vm = yes
+            extra_chardevs = "channel1 channel2 channel3 channel4 channel5 channel6"
+            chardev_backend_channel2 = tcp_socket
+            chardev_backend_channel3 = pty
+            chardev_backend_channel4 = null
+            chardev_backend_channel5 = file
+            chardev_backend_channel6 = ringbuf
+            write_size = 2048
```

Comment:
`ringbuf_write_size_channel6`

Reply:
> `ringbuf_write_size_channel6`

`ringbuf_write_size` can be unified, I will check the parameters `ringbuf_read_size` and `ringbuf_data`.

Reply:
@vivianQizhu updated, please help to review this, thanks.